web/src/components/tasks/BusinessFormation.test.tsx

```diff
@@ -502,10 +552,11 @@ describe("<BusinessFormation />", () => {
           });
         });
 
-        it("displays error message due to failed email validation", async () => {
+        it("displays error message due to failed email validation v2", async () => {
```

Comment:
v2?

Reply:
I was using the similar title for two similar test, I'll update the test name to further clarify.